galeracluster/source/documentation/mysql-wsrep-options.rst

```diff
@@ -1957,6 +1958,48 @@ The contents of the file are subject to change.
    | wsrep_status_file | wsrep-status.json |
    +-------------------+-------------------+
 
+
+
+
+
+.. _`wsrep_sync_server_uuid`:
+.. rst-class:: section-heading
+.. rubric:: ``wsrep_sync_server_uuid``
+
+.. index::
+   pair: Parameters; wsrep_sync_server_uuid
+
+Sets the node to use the server UUID received from the donor node.
+
+.. csv-table::
+   :class: doc-options
+
+   "Command-line Format", "``--wsrep_sync_server_uuid``"
+   "System Variable", "``wsrep_sync_server_uuid``"
+   "Variable Scope", "Global"
+   "Dynamic Variable", "Yes"
+   "Permitted Values", "String"
+   "Default Value", "0"
+   "Initial Version", "MySQL-wsrep 8.0.26-26.8"
+
+Unless this variable is set, the wsrep nodes generate individual server UUIDs, which are used on binlog events, such as rolling schema upgrades, that are not replicated through wsrep. This makes individual node histories incomparable and complicates switching asynchronous slave MASTER between the nodes in the cluster. 
```

Comment:
we probbly should not be saying asynchronous slave MASTER any longer. But I will open up a new PR when we talk about primary/secondary since master/slave is truly a deprecated term